src/main/java/software/aws/neptune/gremlin/GremlinQueryExecutor.java

```diff
@@ -219,7 +216,7 @@ public static void close() {
     }
 
     protected static Client getClient(final GremlinConnectionProperties gremlinConnectionProperties)
-            throws SQLException {
+            throws Exception {
```

Comment:
Why SQLException is replaced by regular Exception?

Reply:
Oh this was a miss from trying different env override methods. Will change back.